app/scripts/lib/snap-keyring/snaps.ts

```diff
@@ -13,7 +13,7 @@ import { SnapKeyringBuilderMessenger } from './types';
  * @returns True if Snap is a preinstalled Snap, false otherwise.
  */
 export function isSnapPreinstalled(snapId: SnapId) {
-  return PREINSTALLED_SNAPS.some((snap) => snap.snapId === snapId);
+  return PREINSTALLED_SNAPS.some((snap) => `npm:${snap}` === snapId);
```

Comment:
Can we add `npm:` to `PREINSTALLED_SNAPS` instead? Right now all preinstalled Snaps use `npm:`, but this may not be the case in the (far) future where we support multiple locations.